process: draft of Coordinating Weight Changes Across Networks (#170)
global-synchronizer-foundation/configs: Super Validator Operational Processes/coordinating-weight-changes-across-networks.md

## Coordinating Weight Changes Across Networks

### Purpose

 - To ensure runtime consistency between onchain Super Validator (SV) weights and the configuration published in `approved-sv-id-values.yaml`. This prevents discrepancies that could cause operational issues and ensures the file remains the authoritative public reference for SV owners, re-onboarding processes, and monitoring.

### Background
- The SV network relies on the consistency between onchain weights and the GitHub configuration file.

> - The configuration file at https://sync.global is used:

Reviewer: ```suggestion
- The configuration file [on github](https://github.com/global-synchronizer-foundation/configs) is used:
```

  - By SV owners to verify weight allocations are correct.
  - During SV re-onboarding (nodes receive the weight defined in the file).

> - Agreement from Operations Call: Adopt Option 3a for synchronization:

Reviewer: ```suggestion
- Agreement from Operations Call: 
```
(no one will remember what option 3a is, and what options 1&2 were...)

  - Keep a single PR if all votes have the same effective date.
  - The node that proposes the weight change is responsible for updating the file.

### Scope
- This process applies to:
  - All SV operators who submit onchain votes to change weights.
  - All networks: MainNet, TestNet, DevNet.

### Process

#### 1. Initiate Onchain Vote

- Propose and pass the weight change vote for the relevant network.
- Ensure the vote is recorded in the governance log.

#### 2. Update GitHub Configuration

- Responsible Party: The SV node that proposed the onchain weight change.
- File to Update: approved-sv-id-values.yaml in configs repo.
- Required Changes: Update the weight values for the relevant SV IDs to match the onchain result.

> - Commit Message Format:`chore(weights): sync with onchain vote [vote ID or link]1

Reviewer: ```suggestion
- Commit Message Format:`chore(weights): sync with onchain vote [vote ID or link]
```
